src/EnergyPlus/PhotovoltaicThermalCollectors.hh

```diff
@@ -50,67 +50,42 @@
 
 // ObjexxFCL Headers
 #include <ObjexxFCL/Array1D.hh>
-#include <ObjexxFCL/Optional.hh>
 
 // EnergyPlus Headers
-#include <EnergyPlus/DataGlobals.hh>
-#include <EnergyPlus/EnergyPlus.hh>
+#include <EnergyPlus/PlantComponent.hh>
 
 namespace EnergyPlus {
 
 namespace PhotovoltaicThermalCollectors {
 
-    // Using/Aliasing
-
-    // Data
-    // MODULE PARAMETER DEFINITIONS:
-    extern int const SimplePVTmodel;
-    extern int const LayerByLayerPVTmodel;
-
-    extern int const ScheduledThermEffic; // mode for thermal efficiency is to use schedule
-    extern int const FixedThermEffic;     // mode for thermal efficiency is to use fixed value
-
-    extern int const LiquidWorkingFluid;
-    extern int const AirWorkingFluid;
-
-    extern int const CalledFromPlantLoopEquipMgr;
-    extern int const CalledFromOutsideAirSystem;
-
-    extern Real64 const SimplePVTWaterSizeFactor; // [ m3/s/m2 ] average of collectors in SolarCollectors.idf
-
-    // DERIVED TYPE DEFINITIONS:
-
-    // MODULE VARIABLE DECLARATIONS:
-    extern Array1D_bool CheckEquipName;
-    extern int NumPVT;              // count of all types of PVT in input file
-    extern int NumSimplePVTPerform; // count of simple PVT performance objects in input file
-
-    // SUBROUTINE SPECIFICATIONS FOR MODULE:
-    // Driver/Manager Routines
-
-    // Utility routines for module
-    // these would be public such as:
-    // PUBLIC  GetPVTIncidentSolarForInternalPVLayer
-    // PUBLIC  GetPVTCellTemp
+    enum struct WorkingFluidEnum
+    {
+        LIQUID,
+        AIR
+    };
 
-    // Types
+    enum struct ThermEfficEnum
+    {
+        SCHEDULED,
+        FIXED
+    };
```

Comment:
`enum struct`, seems so unusual compared to `enum class`; but hey, they are exactly the same, so whatever!